config/sync/field.storage.taxonomy_term.field_report_type.yml

```diff
@@ -0,0 +1,24 @@
+uuid: 1c2c29fe-f3b3-4edc-9258-02ae7dee62c2
+langcode: da
+status: true
+dependencies:
+  module:
+    - options
+    - taxonomy
+id: taxonomy_term.field_report_type
+field_name: field_report_type
+entity_type: taxonomy_term
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: bubble_chart
```

Comment:
It seems a little odd to use something called “chart” for the type of a “report”?